src/screens/landingPageScreens/QuestionsScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+
```

Comment:
CardData is redundand. consider creating an util interface in src/data/utils or smth what you find appropriate
```
interface CardData {
    title: string;
    description: string;
}
```